src/index.ts

```diff
@@ -90,7 +173,7 @@ function successLoader(
   );
 
   // _module.meta is not available inside happypack
-  if (!options.happyPackMode) {
+  if (!options.happyPackMode && loader._module.buildMeta) {
```

Comment:
Oh, I didn't actually mean to commit this change, but it was necessary for testing with our work project. Possibly because we're on Webpack 3.6? Once I added this, everything worked fine.